src/devices/yale.ts

```diff
@@ -195,6 +195,13 @@ const tzLocal = {
 };
 
 const definitions: Definition[] = [
+    {
+        zigbeeModel: ['NF-YRD622 TSDB'],
+        model: 'NF-YRD622 TSDB',
```

Comment:
When googling for this, I don't find any results, please use the model number which is listed on device/product box